### DISCUSSION TITLE
Device update get freezed

### DISCUSSION THREAD

**Discussion by mary-ng5519** [Genymobile/feedback#85] [Genymotion Desktop] [answered]
This is critical bug that freeze the vm, the genymotion desktop keep freezing at "finishing device update ..." in hours without any extra UI reponse or text logs,  giving us negative thought about hidden agenda behind the scence 

![image](https://github.com/Genymobile/feedback/assets/59623591/7391a2a4-133d-46ad-b04c-3f9bd77fa4a8)

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @mary-ng5519,

We are terribly sorry but this board is for feature requests only - it is not suitable for technical support requests (freezing after an update is not a normal behavior and never happened during our QA tests).
Please contact our technical support at https://support.genymotion.com/hc/en-us/

Thank you for your understanding.
Best regards,
